subprojects/packagefiles/opencl-headers/meson.build

```diff
@@ -0,0 +1,16 @@
+project(
+  'OpenCLHeaders',
+  'c',
+  license : 'Apache-2.0',
+  version : '3.0',
```

Comment:
Where does 3.0 come from? Upstream uses `2.2`.

Reply:
Correct. sort of.

README.md says it should be 3.0. My guess is that someone forgot to update CMakeLists.txt.

Reply:
Perhaps. The problem then is that OpenCLHeadersConfigVersion.cmake` sets the version as `set(PACKAGE_VERSION "2.2")` so these will disagree.

Reply:
Replaced with 2.2.